lib/ui/views/login_view.dart

```diff
@@ -174,4 +200,25 @@ class _LoginViewState extends State<LoginView> {
     _focusNode.dispose();
     super.dispose();
   }
+
+  Color getErrorTextColor(String appTheme) {
+    return appTheme == ThemeMode.light.toString()
+        ? Colors.amberAccent
+        : appTheme == ThemeMode.dark.toString()
+            ? Colors.redAccent
+            : Theme.of(context).brightness == Brightness.light
+                ? Colors.amberAccent
+                : Colors.redAccent;
+  }
+
+  Color getButtonColor(String appTheme, BuildContext context, Color lightColor,
+      Color darkcolor) {
+    return appTheme == ThemeMode.light.toString()
+        ? lightColor
+        : appTheme == ThemeMode.dark.toString()
+            ? darkcolor
+            : Theme.of(context).brightness == Brightness.light
+                ? lightColor
+                : darkcolor;
+  }
```

Comment:
```suggestion
  Color get buttonColor => Theme.of(context).brightness == Brightness.light
                ? Colors.white
                : AppTheme.etsLightRed;
```

Same thing of getErrorTextColor 😄